crates/ruma-common/README.md

```diff
@@ -4,4 +4,4 @@
 [![docs.rs page](https://docs.rs/ruma-common/badge.svg)](https://docs.rs/ruma-common/)
 ![license: MIT](https://img.shields.io/crates/l/ruma-common.svg)
 
-Common types for other Ruma crates.
```

Comment:
I don't think removing "other" makes things clearer. We should probably change the description at some point though, probably incorporating the descriptions of the merged-in crates.